__tests__/app.test.js

```diff
@@ -71,14 +71,23 @@ describe('GET', () => {
 });
 
 describe('GET Error handling', () => {
-  test('status: 404 - returns a message for a route that does not exist', () => {
-    return request(app)
-      .get('/api/notARoute')
-      .expect(404)
-      .then((response) => {
-        expect(response.body.message).toBe('not found');
-      });
-  });
+  describe('GET All', () => {
+    test('status: 404 - returns a message for a route that does not exist', () => {
+      return request(app)
+        .get('/api/notARoute')
+        .expect(404)
+        .then((response) => {
+          expect(response.body.message).toBe('not found');
+        });
+    });
+  })
+  describe('GET by Id', () => {
```

Comment:
I'd recommend being a little more descriptive in the describe blocks - what are we getting by ID in these tests?